operations/configuration_management/Version_Control_Service_VCS.md

```diff
@@ -4,43 +4,208 @@ The Version Control Service \(VCS\) includes a web interface for repository mana
+   - The old VCS password, which can be obtained with the following command:
+
+     ```bash
+     ncn# kubectl get secret -n services vcs-user-credentials \
+             --template={{.data.vcs_password}} | base64 --decode
+     ```
+
+1. Enter the existing password (from previous step), new password, and confirmation, and then click `Update Password`.
+1. Now SSH into `ncn-w001` or `ncn-m001`.
+1. Run `git clone https://github.com/Cray-HPE/csm.git`.
+1. Copy the directory `vendor/stash.us.cray.com/scm/shasta-cfg/stable/utils` to the desired working directory.
+1. Change directories to be in the working directory set in the previous step.
+1. Save a local copy of the `customizations.yaml` file.
+
+    ```bash
+    ncn# kubectl get secrets -n loftsman site-init -o jsonpath='{.data.customizations\.yaml}' |
+         base64 -d > customizations.yaml
+    ```
+
+1. Change the password in the `customizations.yaml` file.
+
+   The Gitea `crayvcs` password is stored in the `vcs-user-credentials` Kubernetes Secret in the `services` namespace. This must be updated so that clients which need to make requests can authenticate with the new password.
+
+   In the `customizations.yaml` file, set the values for the `gitea` keys in the `spec.kubernetes.sealed_secrets` field.
+   The value in the data element where the name is `password` needs to be changed to the new Gitea password. The section
+   below will replace the existing sealed secret data in the `customizations.yaml` file.
+
+   For example:
+
+    ```yaml
+          gitea:
+            generate:
+              name: vcs-user-credentials
+              data:
+              - type: static
+                args:
+                  name: vcs_password
+                  value: my_secret_password
+              - type: static
+                args:
+                  name: vcs_username
+                  value: crayvcs
+    ```
+
+1. Upload the modified `customizations.yaml` file to Kubernetes.
+
+   ```bash
+   ncn# kubectl delete secret -n loftsman site-init
+   ncn# kubectl create secret -n loftsman generic site-init --from-file=customizations.yaml
+   ```
+
+1. Encrypt the values after changing the `customizations.yaml` file.
+
+    ```bash
+    ncn# ./secrets-seed-customizations.sh customizations.yaml
+    ```
+
+   If the above command complains that it cannot find `certs/sealed_secrets.crt`, then you can run the following commands to create it:
+
+    ```bash
+    ncn# mkdir -p ../certs &&
+         ./bin/linux/kubeseal --controller-name sealed-secrets --fetch-cert > ../certs/sealed_secrets.crt
+    ```
+
+1. Create a local copy of the `sysmgmt.yaml` file.
+
+    ```bash
+    ncn# kubectl get cm -n loftsman loftsman-sysmgmt -o jsonpath='{.data.manifest\.yaml}'  > sysmgmt.yaml
+    ```
+
+1. Edit the `sysmgmt.yaml` to only include the `gitea` chart and all its current data.
+
+   Example:
+
+    ```yaml
+    apiVersion: manifests/v1beta1
+      metadata:
+        name: sysmgmt
```

Comment:
This will overwrite the loftsman-sysmgmt with the truncated charts. There should also be a step to change this name to something else, like gitea or crayvcs.

You can also take a look at https://github.com/Cray-HPE/docs-csm/blob/main/operations/spire/Restore_Spire_Postgres_without_a_Backup.md. Re-install the Spire Helm Chart Step 3 shows a way to accomplish this step using yq instead of having to manually edit it.

Reply:
@kburns-hpe Made changes as you suggested. Can you please review? Thanks